node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -224,25 +213,28 @@ pub fn server_initializer_collected_params<'a>(
 
     let environment_vcl = EnvironmentVcl::new(&app);
     let commandline_vcl = CommandLineVcl::new(args.to_vec());
-    let (unspecified_vec, specified_vec) = extract_values_vcl_fill_multiconfig_vec(
+    let multiconfig_for_values_extraction = make_new_multi_config(
```

Comment:
Maybe removing the `make` from the function name is a good idea. You might notice this word being extensively used in the functions of test utilities. So, maybe `create_new_multi_config` is a better name here.

Reply:
This function is widely used in production code. I'm going to make a card out of it.